package.json

```diff
@@ -41,6 +41,7 @@
         "@babel/preset-react": "^7.23.3",
         "@babel/preset-typescript": "^7.23.3",
         "@commitlint/cli": "^19.1.0",
+        "@deriv/quill-icons": "^1.19.8",
```

Comment:
@deriv/quill-icons should already be in package.json. Could you please check latest master?

Reply:
yup